pendant/templates/404.html

```diff
@@ -1,12 +1,16 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","layout":{"inherit":true}} -->
-<main class="wp-block-group">
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"72px","bottom":"128px"}}},"layout":{"inherit":true}} -->
+<main class="wp-block-group" style="padding-top:72px;padding-bottom:128px"><!-- wp:heading {"textAlign":"left","level":1,"align":"wide","style":{"typography":{"fontSize":"130px"}}} -->
+<h1 class="alignwide has-text-align-left" id="oops-that-page-can-t-be-found" style="font-size:130px">There's nothing here.</h1>
```

Comment:
We need this translated. The way we do this is via block patterns. Check how Archeo does it. For pendant we already have a block pattern created that you can leverage